macros/stitch/base/stitch_shopify_refunds.sql

```diff
@@ -0,0 +1,152 @@
+{% macro stitch_shopify_refunds() %}
```

Comment:
you're doing two things in this model--
1. handling differences in source data structure
2. applying business logic.

if you separate those two parts out and only do adapter macros for the adjusting of source data structure, you'll have significantly less repeated code.